inc/connector/namespace.php

```diff
@@ -138,11 +138,11 @@ function pre_reschedule_event( $pre, $event ) {
 		]
 	);
 
-	if ( empty( $job[0] ) ) {
+	if ( empty( $job ) || empty ( reset( $job ) ) ) {
```

Comment:
:1234: Avoiding assumption that index is zero rather than job number or some such.